### DISCUSSION TITLE
prometheus-operated service not getting created

### DISCUSSION THREAD

**Discussion by shreyasarani23** [strimzi/strimzi-kafka-operator#4951] [General]
In the documentation they have mentioned Specify a Prometheus server URL (http://prometheus-operated:9090) while adding prometheus as data source .
 but when I do
kubectl get svc -n monitoring i see only
NAME                  TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
grafana               ClusterIP   10.0.35.161   <none>        3000/TCP   96m
prometheus-operator   ClusterIP   None          <none>        8080/TCP   3h51m

Please help

**Comment by shreyasarani23** [comment]
@scholzj  any idea on this??

**Reply by scholzj** [reply]
I do not know, Prometheus is not really my area. Maybe @ppatierno or @sknot-rh might know more. But you should have some service ... it looks to me like you deployed only the operator and not the Prometheus it self. Have you went through [this](https://strimzi.io/docs/operators/latest/full/deploying.html#proc-metrics-deploying-prometheus-str) part? If yes, any errors in the Prometheus operator logs?
